app/controllers/helpers.rb

```diff
@@ -0,0 +1,55 @@
+module Helpers
+	def self.available
+		true
+	end
+
+	def self.can_edit?(session, tweet)
+		#binding.pry
+		if session[:id]
+			tweet.user.id==User.find(session[:id]).id
+		else
+			false
+		end
+	end
+
+	def self.modify_tweet(tweet, params)
+		#binding.pry
+		tweet.content=params[:content]
+		tweet.save
+
+	end
+
+	def self.valid_login(params)
+		!!User.find_by(username: params[:username], password: params[:password])
+	end
+
+	def self.login(user, session)
+		#binding.pry
+		#puts "Loggin in user #{user.id}"
+		session[:id]=user.id
+	end
+
+	def self.login_by_params(params, session)
```

Comment:
A lot of these methods seem redundant.

I would think you only need `logged_in?` and `current_user` really.